alacritty/src/renderer/mod.rs

```diff
@@ -499,6 +493,7 @@ impl Batch {
             bg_g: f32::from(cell.bg.g),
             bg_b: f32::from(cell.bg.b),
             bg_a: cell.bg_alpha,
+            colored: glyph.colored as u8,
```

Comment:
I'm wondering if `colored` is the correct name for this. Since every glyph is technically colored. The only difference is that some glyphs are multi-colored.

Reply:
Good point. Maybe pre_colored, as in pre-colored glyph?

Reply:
Glyphs are monochrome or colored, and you color them in rendering, so technically, `colored` is a fine.